src/monitoring_service/schema.sql

```diff
@@ -3,15 +3,10 @@ CREATE TABLE blockchain (
     receiver                        CHAR(42),
     token_network_registry_address  CHAR(42),
     monitor_contract_address        CHAR(42),
-    latest_known_block              INT
-);
-INSERT INTO blockchain VALUES (
-    NULL,
-    NULL,
-    NULL,
-    NULL,
-    NULL
+    latest_known_block              INT NOT NULL DEFAULT -1
 );
+INSERT INTO blockchain DEFAULT VALUES;
```

Comment:
The default for `CHAR(42)`  is `NULL`?

Reply:
The default for everything is `NULL` unless specified differently.